yilan_zhou/week_01/w1-3_word_guesser.js

```diff
@@ -0,0 +1,36 @@
+var s = "fantastic"; //word to be guessed
+var w=s.split('');
+var guessedW=[];
```

Comment:
It's more annoying to type but I'd like you to use whole words for variable names, generally. You can rename them all before you submit your homework, if you like.